quickfixj-core/src/main/java/quickfix/JdbcStore.java

```diff
@@ -194,7 +200,9 @@ public void reset() throws IOException {
         } catch (SQLException e) {
             throw new IOException(e.getMessage(), e);
         } finally {
-            JdbcUtil.close(sessionID, deleteMessages);
+            if(persistMessages) {
```

Comment:
I think this is not necessary since `JdbcUtil` will check the statement for `null` anyway.
https://github.com/quickfix-j/quickfixj/blob/9df0fe1334b56449a448a985b80df77acae3813b/quickfixj-core/src/main/java/quickfix/JdbcUtil.java#L129-L132

Reply:
Removed check.